src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,51 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Tab } from './types/interface';
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void;
+}
+
+export const Tabs: FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                className={classNames({
+                  'is-active': id === currentTab?.id,
```

Comment:
you always have id in currentTab
```suggestion
                  'is-active': id === currentTab.id,
```